generators/generator-bot-enterprise-assistant/generators/app/templates/botName.dialog

```diff
@@ -330,6 +330,22 @@
             {
               "value": "emulator",
               "actions": [
+                {
+                  "$kind": "Microsoft.IfCondition",
+                  "$designer": {
+                    "id": "QVRnBT"
+                  },
+                  "condition": "=exists(settings.skill) && not(exists(settings.runtimeSettings.skills.allowedCallers))",
```

Comment:
you may also want to check for "" as well. I think if you have something like 

"skills" : { "allowedCallers" : [ "" ] } 

like they might have in a json template it would bypass this still.

Reply:
Otherwise this looks good.